validation/superchain-version_test.go

```diff
@@ -97,6 +97,7 @@ func TestContractVersions(t *testing.T) {
 	}
 
 	for chainID, chain := range OPChains {
+		SkipCheckIfFrontierChain(t, *chain)
```

Comment:
New Issue: Since this skips many chains, we should consider revisiting the list of excluded chain ids hardcoded into each test. Most likely those lists can be shortened or entirely removed.